packages/data/constants.ts

```diff
@@ -179,3 +466,5 @@ export const S3_BUCKET = {
 };
 
 export const DISCORD_URL = 'https://discord.com/invite/9QwXqsyAps';
+
+export const ENS_DOMAIN_URL = 'https://app.ens.domains';
```

Comment:
Looks like https://app.ens.domains/ doesn't support Linea. Should we switch to https://ensfront.dev.linea.build/ specifically for Linea ENS domains ? cc @alainncls @mlallai

Reply:
Yes, you have to switch to Ethereum Goerli to see your domain...  
But Linea ENS doesn't have an equivalent screen (I checked with the team), so we can't redirect there. Let's leave it like that, at least for the moment